### PR TITLE
Add ability to define multiple bread crumbs per route

### PR DESCRIPTION
**Pull request by hbrysiewicz**
Going to try what was attempted in #85 again - to address what was discussed in #19.

I've made it so the attribute key used within a route is `breadCrumbs` as was suggested. This seems clearer than `multipleBreadCrumbs`.

### CODE REVIEW

**Comment by hbrysiewicz** [comment]
Anyway I can get some feedback or movement on this?

**Comment by dguettler** [comment]
Yeah my apologies, I haven’t had much time lately to attend to Issues or
PRs.
I’ll try to get a round of reviews / answers done tonight.

File: addon/components/bread-crumbs.js [hunk 1]
```diff
@@ -95,26 +97,34 @@ export default Component.extend({
...
 
-      let breadCrumb = copy(getWithDefault(route, 'breadCrumb', {
-        title: classify(name)
-      }));
+      const multipleBreadCrumbs = route.get('breadCrumbs');
 
-      if (typeOf(breadCrumb) === 'null') {
-        return;
+      if (multipleBreadCrumbs) {
+        multipleBreadCrumbs.forEach((breadCrumb) => {
```
Comment (dguettler) [inline addon/components/bread-crumbs.js]: @hbrysiewicz any reason for iterating here vs using `breadCrumbs.pushObjects(multipleBreadCrumbs)`

**Comment by alvinvogelzang** [comment]
@hbrysiewicz thanks. Will close https://github.com/poteto/ember-crumbly/pull/85 in favour

**Pull request merged**